client/landing/stepper/declarative-flow/internals/steps-repository/pattern-assembler/style.scss

```diff
@@ -118,6 +118,8 @@ $font-family: "SF Pro Text", $sans;
 			overflow-y: auto;
 			overflow-x: hidden;
 			scrollbar-width: none;
+			// Fix for tooltip position issue
+			padding-bottom: 25px;
```

Comment:
The gap between the list item and add button becomes larger. Could you add `margin-top: 25px` to the last child or the add button as well? thanks!

![image](https://user-images.githubusercontent.com/13596067/224028971-0f5b68c0-5ccd-4cfb-88c3-2ad18c5650bd.png)